src/objects.js

```diff
@@ -1,41 +1,65 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name, age}
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object[property]? true: false;
 };
 
 const isOver65 = person => {
-  // your code here
+  return person.age>65? true:false;
```

Comment:
You don't need the ternary here, you can just return the result of the expression:
```
  return person.age > 65;
```